mlflow/pyfunc/__init__.py

```diff
@@ -286,16 +422,23 @@ def load_model(model_uri, suppress_warnings=True):
                               loading process will be suppressed. If ``False``, these warning
                               messages will be emitted.
     """
-    local_model_path = _download_artifact_from_uri(artifact_uri=model_uri)
-    conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
+    local_path = _download_artifact_from_uri(artifact_uri=model_uri)
```

Comment:
What are the effective changes here?

Reply:
The old code loaded conf for the given flavor in a function call. 
Now we also need MLmodel file to pass to the PyFuncModel, so I parse the MLmodel file here and extract the flavor conf from there.